daemon/images/image_delete.go

```diff
@@ -233,7 +234,7 @@ func (i *ImageService) removeImageRef(ref reference.Named) (reference.Named, err
 	// is an idempotent operation and it's okay if the reference didn't
 	// exist in the first place.
 	_, err := i.referenceStore.Delete(ref)
-
+	logrus.Debugf("Removed reference: %s", ref)
```

Comment:
Changes in this file should probably be reverted now?

Reply:
@thaJeztah I removed the error logs which I already handled in server.go
I only left the debug logs which tells what daemon is doing such as removing references, images, layers